website/home/app/showcase/showcase-client.tsx

```diff
@@ -0,0 +1,389 @@
+                  }}
+                >
+                  {/* Image */}
+                  <div className="aspect-video overflow-hidden bg-gray-100 relative">
+                    {item.image ? (
+                      <Image
+                        src={item.image}
+                        alt={item.title}
+                        width={600}
+                        height={400}
+                        className="w-full h-full object-cover transition-transform duration-500 group-hover:scale-105"
+                      />
+                    ) : (
+                      <div
+                        style={{
+                          background: `linear-gradient(${120 + ((index * 137) % 240)}deg,
+                            hsl(${(index * 137) % 360}, 20%, 85%) 0%,
+                            hsl(${(index * 137 + 60) % 360}, 15%, 88%) 50%,
+                            hsl(${(index * 137 + 120) % 360}, 25%, 83%) 100%)`,
+                        }}
+                        className="w-full h-full"
+                      />
+                    )}
+                    {/* Title overlay */}
+                    <div className="absolute inset-0 flex items-center justify-center p-6">
+                      <h3 className="text-2xl font-bold text-black text-center backdrop-blur-[1px]">
+                        {item.title}
+                      </h3>
+                    </div>
+                    {/* Click indicator */}
+                    {(item.demo || item.github) && (
+                      <div className="absolute top-2 right-2 opacity-0 group-hover:opacity-100 transition-opacity duration-300">
+                        <div className="bg-white/90 backdrop-blur-sm rounded px-2 py-1 flex items-center gap-1 text-xs text-gray-600">
+                          <ExternalLink size={12} />
+                          <span>{item.demo ? "Demo" : "GitHub"}</span>
+                        </div>
+                      </div>
+                    )}
+                  </div>
+
+                  {/* Content */}
+                  <div className="p-6">
+                    <p className="text-gray-600 mb-4">{item.description}</p>
+
+                    {/* Tags */}
+                    <div className="flex flex-wrap gap-2 mb-4">
+                      {item.tags.map((tag) => (
+                        <span
+                          key={tag}
+                          className="text-xs px-2 py-1 bg-gray-100 text-gray-700 rounded-[0px] border border-gray-200"
+                        >
+                          {tag}
+                        </span>
+                      ))}
+                    </div>
+
+                    {/* Links - only show GitHub link if there's also a demo link */}
+                    {item.demo && item.github && (
+                      <div
+                        className="flex gap-3"
+                        onClick={(e) => e.stopPropagation()}
+                      >
+                        <Link
```

Comment:
nit: i think the yellow highlight on the github link is a bit much. something subtler would be nice.